src/components/CryptCards/index.tsx

```diff
@@ -0,0 +1,17 @@
+import {View} from 'react-native';
+import React from 'react';
+import CryptCard from '../CryptCard';
+
+
+
+const CryptCards = () => {
+  return (
```

Comment:
check for all the places where you should use implicit return (if there is nothing before the return statement )